addons/web_editor/static/src/js/editor/odoo-editor/src/utils/utils.js

```diff
@@ -1841,11 +1860,11 @@ export function getState(el, offset, direction, leftCType) {
     if (direction === DIRECTIONS.LEFT) {
         domPath = leftDOMPath(el, offset, reasons);
         inverseDOMPath = rightDOMPath(el, offset);
-        expr = /[^\S\u00A0]$/;
+        expr = /[^\S\u00A0\u0009]$/;
```

Comment:
I'm confused. Why do we have a `nonWhitespacesRegex` const but we write it here as well ? Because of that you had to make multiple changes where you should only have needed one. I suggest to improve this part of the code in this PR.

Reply:
They are all slightly different,  except for the `\S\u00A0\u0009` part : 
![image](https://user-images.githubusercontent.com/3356461/188827105-09343a57-8231-4add-b9d9-856b41d3e230.png)

So I'll just add the common part in a const string and keep the inline regex if it's ok for you.